src/Microsoft.DotNet.Darc/src/Darc/Operations/AddChannelOperation.cs

```diff
@@ -46,12 +46,12 @@ public override async Task<int> ExecuteAsync()
 
                 return Constants.SuccessCode;
             }
-            catch (RestApiException e) when (e.Response.StatusCode == HttpStatusCode.Conflict)
+            catch (RestApiException e) when (e.Response.Status == (int) HttpStatusCode.Conflict)
```

Comment:
Why are we making these changes? Seems unrelated to the code you're adding to Darc.